cmd/rokka/cli/sourceimages.go

```diff
@@ -70,6 +73,51 @@ func copySourceImage(c *rokka.Client, args []string) (interface{}, error) {
 	return nil, c.CopySourceImage(args[0], args[1], args[2])
 }
 
+func copyAllSourceImage(c *rokka.Client, args []string) (interface{}, error) {
+	images := make(chan string)
+	results := make(chan copyall.CopyResult)
+
+	copyAllOptions.SourceOrganization = args[0]
+	copyAllOptions.DestinationOrganization = args[1]
+
+	copyall.StartWorkers(copyAllOptions, rokkaClient, images, results)
+
+	var counterError, counterSuccess int64 = 0, 0
+
+	// get the total count for progress bar
+	listSourceImagesOptions := rokka.ListSourceImagesOptions{}
+	listSourceImagesOptions.Limit = 1
+	res, err := c.ListSourceImages(copyAllOptions.SourceOrganization, listSourceImagesOptions)
+	if err != nil {
+		return nil, err
+	}
+	bar := pb.New(res.Total)
+	bar.ShowSpeed = true
+	if copyAllOptions.NoProgress {
+		bar.NotPrint = true
+	}
+
+	// Scan folders and files
+	go copyall.Scan(copyAllOptions, rokkaClient, images)
+
+	fmt.Fprintf(os.Stderr, "Copying of %d source images started. \n", res.Total)
```

Comment:
you can use `log.Errorf` for that, it does exactly that (see `cmd/rokka/cli/logger.go`). Naming might be a bit bad I guess